src/Microsoft.Extensions.Caching.Memory/project.json

```diff
@@ -23,7 +23,11 @@
     "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {},
+    "net451": {
```

Comment:
Curious why do we need this as netstandard 1.3 includes net451 as per this doc: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md